projects/notebooks/roles/notebooks_dashboard_scale_test/tasks/main.yml

```
@@ -117,43 +131,52 @@
-       -n {{ tester_namespace }}
-       --ignore-not-found
+  command: oc delete cm/dashboard-scale-test-entrypoint -n {{ tester_namespace }} --ignore-not-found
 
 - name: Create the RHODS test entrypoint
-  command:
-    oc create cm dashboard-scale-test-entrypoint
-       "--from-file=entrypoint.sh={{ notebooks_dashboard_scale_test_entrypoint }}"
-       "--from-file=state-signal_barrier.py={{ notebooks_dashboard_scale_test_state_signal_barrier }}"
-       "--from-file=artifacts-exporter.sh={{ notebooks_dashboard_scale_test_s3_artifacts_exporter_sidecar }}"
-       "--from-file={{ notebooks_dashboard_scale_test_test_case_directory }}"
-       -n {{ tester_namespace }}
+  command: oc create cm dashboard-scale-test-entrypoint "--from-file=entrypoint.sh={{ notebooks_dashboard_scale_test_entrypoint }}" "--from-file=state-signal_barrier.py={{ notebooks_dashboard_scale_test_state_signal_barrier }}" "--from-file=artifacts-exporter.sh={{ notebooks_dashboard_scale_test_s3_artifacts_exporter_sidecar }}" "--from-file={{ notebooks_dashboard_scale_test_test_case_directory }}" -n {{ tester_namespace }}
+
 
 - name: Empty the Minio S3 bucket
   shell: |
     oc -c mc -n {{ notebooks_dashboard_scale_test_minio_namespace }} rsh $(oc get pod -lapp=minio -n {{ notebooks_dashboard_scale_test_minio_namespace }} -oname) \
-       mc --config-dir /tmp rm minio/{{ notebooks_dashboard_scale_test_minio_bucket_name }}/ --recursive --force --quiet >/dev/null;
+       mc --config-dir /tmp rm --insecure --recursive --force --quiet minio/{{ notebooks_dashboard_scale_test_minio_bucket_name }}/ >/dev/null;
     oc -c mc -n {{ notebooks_dashboard_scale_test_minio_namespace }} rsh $(oc get pod -lapp=minio -n {{ notebooks_dashboard_scale_test_minio_namespace }} -oname) \
        rm -rf /artifacts/to_export > /dev/null
     oc -c mc -n {{ notebooks_dashboard_scale_test_minio_namespace }} rsh $(oc get pod -lapp=minio -n {{ notebooks_dashboard_scale_test_minio_namespace }} -oname) \
-       mc --config-dir /tmp cp /etc/os-release minio/{{ notebooks_dashboard_scale_test_minio_bucket_name }}; # without it, cp may fail if the bucket is empty
+       mc --config-dir /tmp cp --insecure /etc/os-release minio/{{ notebooks_dashboard_scale_test_minio_bucket_name }}; # without it, cp may fail if the bucket is empty
   failed_when: false
 
+- name: Delete S3 secret minio
+  command: oc delete secret/minio -n {{ tester_namespace }} --ignore-not-found
+
+- name: Create minio Secret
+  shell: set -o pipefail; set -e; SECRET_KEY=$(cat "{{ notebooks_dashboard_scale_test_secret_properties_file }}" | grep user_password | cut -d= -f2); ADMIN_PASSWORD=$(cat "{{ notebooks_dashboard_scale_test_secret_properties_file }}" | grep admin_password | cut -d= -f2); oc create secret generic minio -n "{{ notebooks_dashboard_scale_test_namespace }}" "--from-literal=MINIO_ROOT_PASSWORD=$ADMIN_PASSWORD" "--from-literal=MINIO_S3_ACCESS_KEY=minio" "--from-literal=MINIO_S3_SECRET_KEY=$SECRET_KEY"
+
```

can you reformat this into multiple lines? something like that:
```suggestion
  shell: |
      set -o pipefail; 
      set -e; 
      
      SECRET_KEY=$(cat "{{ notebooks_dashboard_scale_test_secret_properties_file }}" | grep user_password | cut -d= -f2); 
      ADMIN_PASSWORD=$(cat "{{ notebooks_dashboard_scale_test_secret_properties_file }}" | grep admin_password | cut -d= -f2); 
      
      oc create secret generic minio -n "{{ notebooks_dashboard_scale_test_namespace }}" "--from-literal=MINIO_ROOT_PASSWORD=$ADMIN_PASSWORD" "--from-literal=MINIO_S3_ACCESS_KEY=minio" "--from-literal=MINIO_S3_SECRET_KEY=$SECRET_KEY"

```